master/reference/advanced/etcd-rbac/users-and-roles.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Creating Users and Roles in etcd
+---
+
+## Overview
+
+RBAC is *Role* Based Access Control so roles must be setup and also users
```

Comment:
This sentence feels a little out of place, and is already covered in the overview.